.travis.yml

```diff
@@ -51,35 +51,40 @@ matrix:
       env:
         - PLAT=aarch64
         - MB_ML_VER=2014
-    # causes a segfault
-    # - os: linux
-    #  arch: arm64
-    #  env:
-    #    - PLAT=aarch64
-    #    - INTERFACE64=1
-    #    - MB_ML_VER=2014
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
+    - os: linux
+      arch: arm64
+      env:
+        - PLAT=aarch64
+        - INTERFACE64=1
+        - MB_ML_VER=2014
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
```

Comment:
This job is still segfaulting--should we disable for now & perhaps try again when we bump up to `0.3.9` or beyond?

Reply:
yes. Should we report to upstream?

Reply:
If it keeps happening after I rebase the 0.3.9 bump I'll report it yeah